packages/@glimmer/global-context/package.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "@glimmer/global-context",
+  "version": "0.58.0",
+  "repository": "https://github.com/glimmerjs/glimmer-vm/tree/master/packages/@glimmer/global-context",
+  "dependencies": {
+    "@glimmer/env": "^0.1.7"
+  }
```

Adding this will ensure that the first publishing doesn't fail (the default for `npm publish` for scoped packages is private):

```suggestion
  },
  "publishConfig": {
    "access": "public"
  }
```